README.md

```diff
@@ -4,22 +4,25 @@ A set of custom GitHub Actions and reusable workflow used by the Languages Team.
 
 - [Workflows](#workflows)
   - [Buildpacks - Prepare Release](#buildpacks---prepare-release)
+  - [Buildpacks - Prepare Release](#buildpacks---release)
 - [Actions](#actions)
-  - [Generate Buildpack Matrix](#generate-buildpack-matrix)
-  - [Generate Changelog](#generate-changelog)
-  - [Prepare Release](#prepare-release)
-  - [Update Builder](#update-builder)
+  - [Install Languages CLI](#install-languages-cli)
 - [Development](#development)
 
 ## Workflows
 
 ### Buildpacks - Prepare Release
 
 Prepares a buildpack release by:
-* bumping the fixed version
-* updating changelogs
-* generating an aggregate changelog from all the changelogs
-* opening a PR against the repository with the modified files
+- bumping the fixed version
+- updating changelogs
+- generating an aggregate changelog from all the changelogs
+- opening a PR against the repository with the modified files
+
+You can pin to:
+- the [latest release](/releases/latest) version with `latest`
+- a [specific release](/releases) version with `@v{major}.{minor}.{patch}`
+- the development version with `main`
```

Comment:
If I click the links when having opened the `README.md` directly (as opposed to viewing when looking at the GitHub overview UI), they currently 404. 

eg Visit https://github.com/heroku/languages-github-actions/blob/06ac54789d81ade15ace09a6e1fa18ae1860b659/README.md and click the links.

I think the only way to avoid that is having to include the full URL sadly.

Also, currently some of the examples are prefixed with `@` but some aren't - I think for clarity/consistency we should prefix them all with `@`.

```suggestion
- the [latest release](https://github.com/heroku/languages-github-actions/releases/latest) version with `@latest`
- a [specific release](https://github.com/heroku/languages-github-actions/releases) version with `@v{major}.{minor}.{patch}`
- the development version with `@main`
```